smart-server/src/test/java/net/obvj/smart/conf/AgentsXmlTest.java

```diff
@@ -134,7 +140,7 @@ public void testLoadXmlWithTwoAgents()
     @Test
     public void testLoadSchemaFileWithInvalidFile()
     {
-        assertException(AgentConfigurationException.class, null, SAXException.class,
-                () -> AgentsXml.loadSchemaFile("testAgents/invalidSchema.xsd"));
+        assertThat(() -> AgentsXml.loadSchemaFile("testAgents/invalidSchema.xsd"),
+                throwsException(AgentConfigurationException.class).withCause(SAXParseException.class));
```

Comment:
An issue was open at net.obvj.junit-utils (issue/0009) to fix this inconsistency regarding the SAXException.class.